README.md

```diff
@@ -1,16 +1,43 @@
 # TVM Project Homepage
 
-## Prereq
+## Dependencies
 
-- Install the jekyll and bundler gems `gem install bundler jekyll`
+1. Install ruby
 
+    ```bash
+    # ubuntu
```

Comment:
want to state the ubuntu release here? I find sometimes this changes or proves helpful if you need to search for e.g. "why did bundler get installed through package foo in 20.04?"